topics/synthetic-biology/tutorials/basic_assembly_analysis/tutorial.md

```diff
@@ -0,0 +1,225 @@
+- How to generate scripts enabling the automatic build of plasmids ?
+objectives:
+- Annotate the SBML pathway using Selenzy tool.
+- Design plasmids encoding predicted pathways using rpbasicdesign tool.
+- Generate dnabot scripts enabling the automatic build of plasmids.
+time_estimation: 15M
+key_points:
+- Annotation of SBML pathway using Selenzyme
+- Generating Constructs using rpbasicdesign
+- Generating python scripts using dnabot
+contributors:
+- kenza12
+- tduigou
+- breakthewall
+- guillaume-gricourt
+- ioanagry
+- jfaulon
+
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+Synthetic biology is a novel engineering discipline which requires computational tools for the design of metabolic pathways for the production of chemicals such as SynBioCAD portal which is the first Galaxy set of tools for synthetic biology and metabolic engineering ({% cite Hrisson2022 %}). 
+
+In this tutorial, we will use a set of tools from the **Genetic Design - BASIC Assembly Workflow** (https://galaxy-synbiocad.org) which will enable you to design plasmids implementing metabolic pathways for the bioproduction of lycopene in _E.coli_ (one of the preferred host cell for microbial biochemicals production).
+
+Lycopene is a potent antioxidant and has been widely used in the fields of pharmaceuticals, nutraceuticals, and cosmetics. It's widely found in fruits including tomato, watermelon, guava, and papaya but the extraction method of lycopene from these natural sources is expensive, complicated and cannot match the large market demand.
+
+To address this demand, synthetic biology and metabolic engineering have been employed to develop microbial cell factories (e.g. _E.coli_ strains) for lycopene production.
+
+To design plasmids encoding lycopene bioproducing pathways, we will use the BASIC assembly method (as described in {% cite Storch2015 %}) which relies on orthogonal linkers and type IIs restriction enzyme cleavage to provide a robust and accurate assembly of DNA parts into plasmid constructs. From these construct definition, the workflow will generate scripts enabling the automatic build of the plasmids as well as  the transformation of strains using an Opentrons liquid handler robot. After downloading these scripts onto a computer connected to an Opentrons, one can perform the automated construction of the plasmids at the bench.
+
+The workflow scheme we will use is shown below. First, we will run the steps of this workflow individually so as not to neglect the understanding of the intermediate steps as well. Then, we will run the workflow automatically so that it itself retrieves the outputs from the previous step and gives them as input to the next tool.
+
+![Genetic Design - BASIC Assembly Workflow](../../images/basic_assembly_workflow.png)
+
+Note that a video tutorial is also available for this workflow : [Galaxy BASIC Assembly Workflow Tutorial ](https://youtu.be/it_JTVdvaRo)

```

Comment:
A video, nice! Would you be ok with adding that to the [GTN Video Library](https://gallantries.github.io/video-library/)? That way the video will be automatically linked from the top of the tutorials, and from the topic page (under "recordings"). And it would also make it easier to create online training events around your tutorials.

We would re-upload this tutorial to the GalaxyProject YouTube account, and ask you to correct the auto-generated captions that YouTube generates, so that we can provide quality subtitles and increase accessibility for our users.  We will copy the autogenerated subs to a Google Doc for you and others to fix, usually its just a matter of punctuation and fixing technical terms)

Reply:
I submitted the video to the GTN Video Library 2 weeks ago. I don't know if you were able to analyze it ? Thanks.

Reply:
Hi @shiltemann , thank you for analyzing the video !

Since the video will appear from the top of the tutorial, I deleted the old video link in the tutorial.

About the speaker, no it's not me. He's one of the tutorial contributors:

```
breakthewall:
    name: Joan Hérisson
    email: joan.herisson@univ-evry.fr
    orcid: 0000-0001-9741-0847
    gitter: breakthewall
    linkedin: joanherisson
    joined: 2022-03
```
Name of institute: Université d'Évry / Paris-Saclay, CNRS, CEA, INRAE
Webpage: http://www.univ-evry.fr/

He's an employee of the University of Évry / Paris-Saclay supervised by CNRS and CEA. He's also working in collaboration with INRAE in this project. I will let you pick the needed informations.

Thanks again.

![evry_logo](https://user-images.githubusercontent.com/17780887/168045191-8a49d2d2-6831-4967-aac3-162402c201c6.png)
![Logo-INRAE_Quadri-web](https://user-images.githubusercontent.com/17780887/168045248-6c6e521d-aba6-4732-8b78-f4b248fc7751.jpg)
![CNRS](https://user-images.githubusercontent.com/17780887/168045262-58dd444b-f36e-4ce5-9d9a-fd7a4c45e716.jpg)
![CEA_logo_nouveau](https://user-images.githubusercontent.com/17780887/168045273-dd28eaa9-16f8-4aae-a4d2-beb641fd9018.svg)

Reply:
Perfect, thanks!

Here is a link to the video captions as generated by YouTube, if you could go over it fix where needed, that would be great

https://docs.google.com/document/d/1edwDeJtySnO2d9nLbwPBpoUeBdEYE-9sLKhCJiEJk3k/edit?usp=sharing

Reply:
Ok, we fixed the video captions, thank you !!